packages/nx/src/command-line/graph/graph.ts

```diff
@@ -752,6 +834,130 @@ function createTaskId(
   }
 }
 
+async function getExpandedTaskInputs(
+  taskId: string
+): Promise<Record<string, string[]>> {
+  const [project] = taskId.split(':');
+  const taskGraphResponse = await createTaskGraphClientResponse(false);
+
+  const allWorkspaceFiles = await allFileData();
+
+  const inputs = taskGraphResponse.plans[taskId];
+  if (inputs) {
+    return expandInputs(
+      inputs,
+      currentDepGraphClientResponse.projects.find((p) => p.name === project),
+      allWorkspaceFiles,
+      currentDepGraphClientResponse
+    );
+  }
+  return {};
+}
+
+function expandInputs(
+  inputs: string[],
+  project: ProjectGraphProjectNode,
+  allWorkspaceFiles: FileData[],
+  depGraphClientResponse: ProjectGraphClientResponse
+): Record<string, string[]> {
```

Comment:
This should be replaced by the rust functionality that is going to be built out soon. But for now it's fine to have here. I'll replace this when the new function is ready.